stable/goldilocks/templates/vpa-uninstall-hook.yaml

```diff
@@ -1,65 +1,65 @@
-{{- if .Values.upgradeVPA }}
+{{- if or (eq .Values.uninstallVPA true) (eq .Values.reinstallVPA true) }}
```

Comment:
Maybe simplify to `if or .Values.uninstallVPA .Values.resinstallVPA`

Reply:
Oh sure, why not.  I was being lazy and copying from somewhere ;-)